.github/workflows/test.sh

```diff
@@ -9,36 +9,41 @@ conda remove -q -n test-env --all || true
 # Create and activate conda environment for this test
 BINDING=$(echo "$1" | tr '[:lower:]' '[:upper:]')
 QT_VERSION_VAR=${BINDING}_QT_VERSION
-# pytest-qt >=4 doesn't support Qt >=5.9
+# pytest-qt >=4 doesn't support Qt <=5.9
 if [ "${!QT_VERSION_VAR:0:3}" = "5.9" ]; then PYTESTQT_VERSION="=3.3.0"; PYTEST_VERSION=">=6,!=7.0.0,!=7.0.1,<7.2.0"; fi
```

Comment:
```suggestion

# pytest-qt >=4 doesn't support Qt <=5.9
if [ "${!QT_VERSION_VAR:0:3}" = "5.9" ]; then
    PYTESTQT_VERSION="=3.3.0"
    PYTEST_VERSION=">=6,!=7.0.0,!=7.0.1,<7.2.0"
fi

```

No need for this `if` to be in a single line, which makes it hard to read.

Reply:
Yeah, I'm not sure why it was that way originally; it was added at some point after I created the original script